projects/hslayers/src/css/_functions-override.scss

```diff
@@ -112,6 +112,17 @@
   // @return (h: $h, s: $s, l: $l, a: $a);
 }
 
+@function getUnit($value) {
```

Comment:
What happens here?

Reply:
@raitisbe Basically there was a warning when building the  app about invalid value for some styling functions, Filip dealt with it! :)